packages/ERTP/test/swingsetTests/ertpService/bootstrap-ertp-service-upgrade.js

```diff
@@ -64,7 +66,7 @@ export const buildRootObject = () => {
 
     upgradeV2: async () => {
       const bcap = await E(vatAdmin).getNamedBundleCap('ertpService');
-      await E(ertpAdmin).upgrade(bcap);
+      await E(ertpAdmin).restartVat(bcap);
```

Comment:
I think the method name should be `restart`, not `restartVat()`, because the admin facet is already scoped to a particular vat.